kzz7yh-f18593/cod-ve09v2_kzz7yh-f18593.xml

```diff
@@ -216,7 +216,7 @@
           </p>-->
         <p xml:id="kzz7yh-f18593-d1e106">
           Disti VI. 
-          <lb ed="#V" n="77"/>Reterea sciri oporte etc. Superius 
+          <lb ed="#V" n="77"/>PReterea sciri oportet etc. Superius 
```

Comment:
Why is this abbreviation "opportet" and not "opportum"?

Reply:
![image](https://github.com/scta-texts/kzz7yh/assets/1146685/897bbc73-3e61-4dc9-85af-d387d34eafb3)

The little sidewise "m" you see at the end can be an "m" but it can also be for "et". you will see this symbol used in words like "pat-et", "lic-et" and here "oport-et"